corehq/apps/app_manager/models.py

```diff
@@ -3573,8 +3573,8 @@ def add_insert_form(self, from_module, form, index=None, with_source=False):
                 )
 
                 if from_module.parent_select.active:
-                    app = self.get_app()
-                    select_chain = get_select_chain(app, from_module, include_self=False)
+                    from_app = from_module.get_app()
+                    select_chain = get_select_chain(from_app, from_module, include_self=False)
```

Comment:
nit: i am not sure how much adding `from_` adds value to app here. You could just change to
`app = from_module.get_app()`
Instead add documentation to the method that makes it clear that from_module can be within the same app or a different app as well considering that's not obvious.